projects/storefrontapp-e2e-cypress/cypress/e2e/regression/checkout/express-checkout.e2e.cy.ts

```diff
@@ -1,11 +1,19 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 import { CheckoutConfig } from '@spartacus/storefront';
 import * as checkout from '../../../helpers/checkout-flow';
 import * as expressCheckout from '../../../helpers/express-checkout';
 import { viewportContext } from '../../../helpers/viewport-context';
 import { clearAllStorage } from '../../../support/utils/clear-all-storage';
+import { isolateTestsBefore } from '../../../support/utils/test-isolation';
 
-context('Express checkout', () => {
+context('Express checkout', { testIsolation: false }, () => {
   viewportContext(['mobile', 'desktop'], () => {
+    isolateTestsBefore();
```

Comment:
Clearing localStorage and cookies only before the 'context' block, not before each 'it'.
Reason: cookie 'checkout express true' (set in 'before' hook) needs to exist all along 'it' blocks